UI/src/components/molecules/RipaStopReason.vue

```diff
@@ -157,6 +157,48 @@
             :rules="explanationRules"
             @blur="handlePiiCheck($event)"
           ></ripa-text-input>
+          <template v-if="model.stopType === 'Vehicular'">
```

Comment:
Can you add a "&& model.stopVersion === 2" just to make sure it doesn't render for V1 stops?